core/src/scene/sceneLoader.h

```diff
@@ -27,6 +28,7 @@ struct MaterialTexture;
 struct Filter;
 
 using Mixes = std::vector<YAML::Node>;
+using UniqueStyles = std::unordered_set<std::string>;
```

Comment:
Sure... will just make these function signatures huge.
And because of that will use the new way of defining function signatures.